src/styles/blocks/card/blockscard/starReviewsPrice.scss

```diff
@@ -0,0 +1,78 @@
+.card__review {
+  display: flex;
+  align-items: flex-end;
+  justify-content: flex-end;
+  max-width: 166px;
+  margin-block: 8px;
+  gap: 17px;
+
+  &-amount {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-end;
+    align-items: flex-end;
+    height: 16px;
+    width: 53px;
+    margin: 6px 0 10px;
```

Comment:
Avoid using shorthand properties like `margin` for setting only one or two sides. It's better to use specific properties like `margin-top` or `margin-bottom`. It makes the code more readable and maintainable.